### PR TITLE
Add ByteBuf reversed byte extension functions

### PR DESCRIPTION
**Pull request by Tomm0017**
Revision #191 writes the appearance buffer content as reversed bytes.

### CODE REVIEW

**Comment by Bartvhelvert** [comment]
Looks good, although the get, set and read methods should also be included.

**Comment by Tomm0017** [comment]
Included them, let me know if there's anything missing

File: src/main/kotlin/io/guthix/buffer/ByteBuf.kt [hunk 1]
```diff
@@ -142,6 +142,13 @@ public fun ByteBuf.getBytesReversedAdd(index: Int, dest: ByteArray): ByteArray =
     getBytes(index, this)
 }.map { (it - HALF_BYTE).toByte() }.reversed().toByteArray()
 
+public fun ByteBuf.getBytesReversed(index: Int, length: Int): ByteArray =
+        getBytesReversed(index, ByteArray(length))
```
Comment (Bartvhelvert) [inline src/main/kotlin/io/guthix/buffer/ByteBuf.kt:146]: The indentation is inconsistent here

**Review by Bartvhelvert — CHANGES_REQUESTED** [review]
Thanks! Other than the formatting it looks good.

**Review by Bartvhelvert — APPROVED** [review]

**Pull request merged**